### PR TITLE
Update full_cases.txt

### PR DESCRIPTION
**Pull request by muccry**

### CODE REVIEW

**Comment by muccry** [comment]
1684X:
name,prec, shape,gops, time(ms),mac utilization, cpu usage, ddr_utilization
unet,FP32,1x3x640x640,396.200,458.477,43.21%,15.00%,13.38%
unet,INT8,1x3x640x640,396.200,25.494,48.57%,6.00%,20.56%
unet,INT8,4x3x640x640,1584.800,91.556,54.09%,10.00%,22.46%

1684:
name,prec, shape, gops, time(ms), mac utilization, cpu usage, ddr_utilization
unet,FP32,1x3x640x640,396.200,330.647,N/A,47.90%,N/A
unet,INT8,1x3x640x640,396.200,132.070,N/A,8.00%,N/A
unet,INT8,4x3x640x640,1584.800,150.686,N/A,10.00%,N/A

sdk:
tpu-nntc V3.1.2-8b15965f-220926
sophon-driver_0.4.1
sophon-libsophon_0.4.1

**Comment by muccry** [comment]
add MobileNet-SSD:

1684X:
name,opt,dyn,prec,shape,gops,time(ms),mac_utilization,cpu_usage,ddr_utilization
MobileNet-SSD,1,False,FP32,1x3x300x300,2.300,2.000,57.51%,9.00%,0.32%
MobileNet-SSD,1,False,FP32,4x3x300x300,9.200,7.040,65.34%,8.00%,0.36%
MobileNet-SSD,1,False,FP32,8x3x300x300,18.400,13.727,67.02%,7.00%,0.38%
MobileNet-SSD,1,False,FP32,16x3x300x300,36.800,26.945,68.29%,8.00%,0.39%
MobileNet-SSD,2,True,FP32,1x3x300x300,2.300,2.000,57.50%,10.00%,0.32%
MobileNet-SSD,2,True,FP32,4x3x300x300,9.200,7.023,65.50%,7.00%,0.36%
MobileNet-SSD,2,True,FP32,8x3x300x300,18.400,13.730,67.00%,7.00%,0.37%
MobileNet-SSD,2,True,FP32,16x3x300x300,36.800,26.945,68.29%,9.00%,0.38%
MobileNet-SSD,1,False,INT8,1x3x300x300,2.300,1.050,6.85%,8.00%,0.56%
MobileNet-SSD,1,False,INT8,4x3x300x300,9.200,3.421,8.40%,5.00%,0.69%
MobileNet-SSD,1,False,INT8,8x3x300x300,18.400,6.525,8.81%,5.00%,0.73%
MobileNet-SSD,1,False,INT8,16x3x300x300,36.800,12.772,9.00%,3.00%,0.74%
MobileNet-SSD,2,True,INT8,1x3x300x300,2.300,1.052,6.83%,8.00%,0.56%
MobileNet-SSD,2,True,INT8,4x3x300x300,9.200,3.427,8.39%,5.00%,0.69%
MobileNet-SSD,2,True,INT8,8x3x300x300,18.400,6.503,8.84%,5.00%,0.73%
MobileNet-SSD,2,True,INT8,16x3x300x300,36.800,12.776,9.00%,4.00%,0.74%


1684：
name,prec,opt,dyn,shape,gops,time(ms),mac_utilization,cpu_usage,ddr_utilization
MobileNet-SSD,FP32,1,False,1x3x300x300,2.300,3.304,31.64%,8.00%,0.39%
MobileNet-SSD,FP32,1,False,4x3x300x300,9.200,10.689,39.12%,10.00%,0.49%
MobileNet-SSD,FP32,1,False,8x3x300x300,18.400,20.659,40.48%,13.00%,0.51%
MobileNet-SSD,FP32,1,False,16x3x300x300,36.800,40.728,41.07%,20.00%,0.51%
MobileNet-SSD,FP32,2,True,1x3x300x300,2.300,3.311,31.57%,9.00%,0.41%
MobileNet-SSD,FP32,2,True,4x3x300x300,9.200,10.683,39.15%,9.00%,0.49%
MobileNet-SSD,FP32,2,True,8x3x300x300,18.400,20.658,40.49%,14.00%,0.51%
MobileNet-SSD,FP32,2,True,16x3x300x300,36.800,40.751,41.05%,19.00%,0.51%
MobileNet-SSD,INT8,1,False,1x3x300x300,2.300,3.084,4.24%,3.00%,0.46%
MobileNet-SSD,INT8,1,False,4x3x300x300,9.200,6.291,8.31%,3.00%,0.89%
MobileNet-SSD,INT8,1,False,8x3x300x300,18.400,11.908,8.78%,3.00%,0.95%
MobileNet-SSD,INT8,1,False,16x3x300x300,36.800,23.120,9.04%,3.00%,0.96%
MobileNet-SSD,INT8,2,True,1x3x300x300,2.300,3.085,4.24%,4.00%,0.46%
MobileNet-SSD,INT8,2,True,4x3x300x300,9.200,6.290,8.31%,3.00%,0.89%
MobileNet-SSD,INT8,2,True,8x3x300x300,18.400,11.900,8.79%,3.00%,0.95%
MobileNet-SSD,INT8,2,True,16x3x300x300,36.800,23.105,9.05%,3.00%,0.97%

sdk:
tpu-nntc V3.1.2-8b15965f-220926
sophon-driver_0.4.1
sophon-libsophon_0.4.1

File: full_cases.txt [hunk 1]
```diff
@@ -60,3 +60,5 @@ vision/RNN/GRU
 vision/tracking/GOTURN-Caffe
 vision/tracking/MDNet-Torch
 vision/tracking/SiamMask-Torch
+vision/segmentation/unet
+vision/classification/MobileNet-SSD
```
Comment (Jeffwhen) [inline full_cases.txt:64]: SSD 应该放到 detection

File: vision/classification/MobileNet-SSD/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+## Dataset
+
+[ImageNet (VOC2012train)](http://host.robots.ox.ac.uk/pascal/VOC/voc2012/VOCtrainval_11-May-2012.tar).
+[ImageNet (VOC2007train)](http://host.robots.ox.ac.uk/pascal/VOC/voc2007/VOCtrainval_06-Nov-2007.tar).
+[ImageNet (VOC2007test)](http://host.robots.ox.ac.uk/pascal/VOC/voc2007/VOCtest_06-Nov-2007.tar).
+[ImageNet (MS COCO)](https://cocodataset.org/).
+
+## References
+
+* [weiliu89/caffe/tree/ssd](https://github.com/weiliu89/caffe/tree/ssd)
```
Comment (Jeffwhen) [inline vision/classification/MobileNet-SSD/README.md:24]:
```
[weiliu89/caffe](https://github.com/weiliu89/caffe/tree/ssd)
```

File: vision/classification/MobileNet-SSD/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+| Model         | Download                                                                       | mpa  |
+|---------------|:-------------------------------------------------------------------------------|:-----|
+| MobileNet-SSD | [model](MobileNetSSD_deploy.prototxt) [weight](MobileNetSSD_deploy.caffemodel) | 72.5 |
+
+## Dataset
+
+[ImageNet (VOC2012train)](http://host.robots.ox.ac.uk/pascal/VOC/voc2012/VOCtrainval_11-May-2012.tar).
+[ImageNet (VOC2007train)](http://host.robots.ox.ac.uk/pascal/VOC/voc2007/VOCtrainval_06-Nov-2007.tar).
+[ImageNet (VOC2007test)](http://host.robots.ox.ac.uk/pascal/VOC/voc2007/VOCtest_06-Nov-2007.tar).
+[ImageNet (MS COCO)](https://cocodataset.org/).
```
Comment (Jeffwhen) [inline vision/classification/MobileNet-SSD/README.md:20]: 写成一个列表：
```
+ [ImageNet (VOC2012train)](http://host.robots.ox.ac.uk/pascal/VOC/voc2012/VOCtrainval_11-May-2012.tar).
+ [ImageNet (VOC2007train)](http://host.robots.ox.ac.uk/pascal/VOC/voc2007/VOCtrainval_06-Nov-2007.tar).
+ [ImageNet (VOC2007test)](http://host.robots.ox.ac.uk/pascal/VOC/voc2007/VOCtest_06-Nov-2007.tar).
+ [ImageNet (MS COCO)](https://cocodataset.org/).
```

File: vision/segmentation/unet/config.yaml [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  int8_weight: $(workdir)/$(name)_bmpaddle.int8umodel
+
+  bmnetu_batch_sizes: [1, 4]
+  bmnetu_options:
+    --model $(int8_model)
+    --weight $(int8_weight)
+    --target $(target)
+    --enable_profile True
+
+BM1684:
```
Comment (Jeffwhen) [inline vision/segmentation/unet/config.yaml:39]: BM1684 和 BM1684X 的选项没有区别？

**Review by Jeffwhen — CHANGES_REQUESTED** [review]

**Pull request closed without merging**